### PR TITLE
Add List 3D Nodes query parameters

### PR DESCRIPTION
**Pull request by haakonflatval-cognite**

### CODE REVIEW

**Comment by haakonflatval-cognite** [comment]
The reason for the changes to `cognite/client/_api_client.py`, is that the function `get_partition()` previously only supported getting partitions through the HTTP `POST` method, while `List 3D Nodes` expects `GET`.

File: cognite/client/_api_client.py [hunk 1]
```diff
@@ -446,14 +448,22 @@ def get_partition(partition):
...
+                    body = {
+                        "filter": filter or {},
+                        "limit": self._LIST_LIMIT,
+                        "cursor": next_cursor,
+                        "partition": partition,
+                        **(other_params or {}),
+                    }
+                    res = self._post(url_path=resource_path + "/list", json=body, headers=headers)
+                elif method == "GET":
+                    params = filter.copy()
```
Comment (sanderland) [inline cognite/client/_api_client.py:461]: params = {**filter.copy(),
              "limit": ...
              "cursor": ...
                **other_params}

**Review by sanderland — CHANGES_REQUESTED** [review]
one refactoring suggestion, and would like a test

**Review by me-ydv-5 — COMMENTED** [review]
Please also add changelog and update version.

**Comment by haakonflatval-cognite** [comment]
The test is currently skipped, just as the rest of the revision tests. This is because we don't have a setup for creating a revision with a 3d file. Adding such a setup is out of scope of this PR.

File: cognite/client/_api/three_d.py [hunk 2]
```diff
@@ -378,7 +378,14 @@ def update_thumbnail(self, model_id: int, revision_id: int, file_id: int) -> Non
         self._post(resource_path, json=body)
 
     def list_nodes(
-        self, model_id: int, revision_id: int, node_id: int = None, depth: int = None, limit: int = 25
+        self,
+        model_id: int,
+        revision_id: int,
+        node_id: int = None,
+        depth: int = None,
+        limit: int = 25,
```
Comment (sanderland) [inline cognite/client/_api/three_d.py:386]: I think we tend to keep limit as the last parameter, as it makes the docs a little more readable

**Review by sanderland — CHANGES_REQUESTED** [review]
tiny change i missed before

**Review by sanderland — APPROVED** [review]

**Pull request merged**